src/main/java/com/epam/grid/engine/provider/job/sge/SgeJobProvider.java

```diff
@@ -236,85 +255,50 @@ private String[] makeQdelCommand(final DeleteJobFilter filter) {
      * @param deleteJobFilter User-defined conditions.
      */
     private void validateDeleteRequest(final DeleteJobFilter deleteJobFilter) {
-        if (!StringUtils.hasText(deleteJobFilter.getUser()) && deleteJobFilter.getId() == null) {
+        if (!StringUtils.hasText(deleteJobFilter.getUser()) && CollectionUtils.isEmpty(deleteJobFilter.getIds())) {
```

Comment:
how system will behave if we pass `owner` as well as `ids`, but ids weren't launched from `owner`? I believe it is better to check it here to, and do not allow run request if both things are specified

Reply:
I also believe that such check could be moved to service layer to reuse this code

Reply:
done